ibazel/command/command.go

```diff
@@ -41,37 +37,18 @@ type Command interface {
 
 // start will be called by most implementations since this logic is extremely
 // common.
-func start(b bazel.Bazel, target string, args []string) (*bytes.Buffer, *exec.Cmd) {
-	var filePattern strings.Builder
-	filePattern.WriteString("bazel_script_path")
-	if runtime.GOOS == "windows" {
-		filePattern.WriteString(".bat")
-	}
-	
-	tmpfile, err := ioutil.TempFile("", filePattern.String())
-	if err != nil {
-		fmt.Print(err)
-	}
-	// Close the file so bazel can write over it
-	if err := tmpfile.Close(); err != nil {
-		fmt.Print(err)
-	}
-
-	// Start by building the binary
-	_, outputBuffer, _ := b.Run("--script_path="+tmpfile.Name(), target)
-
-	runScriptPath := tmpfile.Name()
-
-	// Now that we have built the target, construct a executable form of it for
-	// execution in a go routine.
-	cmd := execCommand(runScriptPath, args...)
+func start(target string, args []string) (*bytes.Buffer, *exec.Cmd) {
```

Comment:
I think you should continue to use the bazel abstraction here

Reply:
I was considering that but then it seemed the way the process is called is a bit different as ctrl+c did not work properly to interrupt the process. So I thought that maybe there is a reason why the binary was called with its own subprocess call. But can look into it in more detail to understand what's going on.